### PR TITLE
Finish test function and update README

### PR DESCRIPTION
**Pull request by rq1995**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,32 +99,28 @@ BIC = n*log(residual sum of squares/n) + K*log(n)
...
 
-* **model**: default 'linear' | 'logistic' | 'ridge' | 'lasso' | 'elasticnet'
-  * Method applied to the model
-
 **Return:**
-* BIC score of the model: int
+* bic_score: int
+  * BIC score of the model
 
 ### Mallow's C_p
```
Comment (hadinh1306) [inline README.md:126]: Can you add math function of mallow?

File: tests/testthat/testthat_mallows.R [hunk 2]
```diff
@@ -1 +1,26 @@
...
+#library(regscoreR)
+
+context("Testing mallows")
+
+# mallows
+test_that("mallows() returns Mallow's Cp of the model", {
+  # expected outputs:
+  expect_equal(round(mallow([1,2,3,4],[5,6,7,8],[1,2,3,5],3,2),5), 1.09375)
+})
+
```
Comment (hadinh1306) [inline tests/testthat/testthat_mallows.R]: you wanna add cases with negative and floating numbers?

**Review by hadinh1306 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rq1995) [reply README.md:126]: I already have math function like `C_p = (SSE_p/MSE) - (n - 2p)`. Or you mean something else?

File: tests/testthat/testthat_mallows.R (hunk 2 above)
Comment (rq1995) [reply tests/testthat/testthat_mallows.R]: you mean test the input is floating and negative numbers or not? or add cases with the negative output?

File: README.md (hunk 1 above)
Comment (hadinh1306) [reply README.md:126]: Yea that's the function. I don't see it in your pull request though

File: tests/testthat/testthat_mallows.R (hunk 2 above)
Comment (hadinh1306) [reply tests/testthat/testthat_mallows.R]: I mean test the input with floating and negative numbers in y and y_pred.

Comment (hadinh1306) [reply tests/testthat/testthat_mallows.R]: And maybe input with 0 in it. I don't think it matters much in our case, but maybe it does when we test with our function.

File: README.md (hunk 1 above)
Comment (rq1995) [reply README.md:126]: We already included it in current version. I did not delete it. You can see it in current repo.

**Review by hadinh1306 — APPROVED** [review]

**Pull request merged**